### ISSUE TITLE
A11y_MySQLextensionforAzureDataStudio_Connect to a MySQL Server_I want to connect to a MySQL Server by providing server and authentication details so that I can manage it_VoiceOver: Voiceover is not announcing the "Connection Details" section name when voiceover focus is on the first field of section.

### ISSUE THREAD

**Issue by Raghu-addepalli** [microsoft/azuredatastudio-mysql#197]
"[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #GH-MySQLextensionforAzureDataStudio-MAC-Mar23; #MySQL extension for Azure Data Studio; #Screenreader; #VoiceOver; #Benchmark; #WCAG1.3.1; #Info and Relationships; #Benchmark; #FTP; #MAC; #AILimited; #Element:Dropdown; #A11yWCAG2.1;

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."**

### Environment and OS details:
Application Name: Azure Data Studio
Application Version: 14.2.0
Operating system: macOS Ventura (Version 13.3)
Screen Reader: Voiceover.

### Pre-Requisite:

1. Install Azure data studio From here https://learn.microsoft.com/en-us/sql/azure-data-studio/download-azure-data-studio?view=sql-server-ver16&tabs=redhat-install%2Credhat-uninstall
2. Once installation done , Install MySQL Extension in Azure data studio
3. Tab Till new connection and press enter
4. Enter a valid server name username and password and press connect

### Repro Steps:

1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Hover over the 'Servers' header and select the 'New Connection' icon.
4. Select server type as MySQL.
5. Turn on voiceover.
6. Tab to "Connection type" dropdown present in the "Connection Details" section.
7. Now check that Voice over is announcing the section name or not.
### Actual Result:
Voiceover is not announcing the "Connection Details" section name when voiceover focus is on the first field of section.
### Expected Result:
Voiceover is not announcing the "Connection Details" section name when voiceover focus is on the first field of section.
### User Impact:
Screen Reader users will not be able to understand in which section the focus is there if the section name is not announced.
### Attachment:

https://user-images.githubusercontent.com/91586743/229107313-75aa7923-cfea-4526-bdc0-4afd39d765b1.mp4

**Comment by nitishsharma25** [comment]
This issue is external to MySQL extension as it is a common feature of Azure Data Studio.

**Comment by Raghu-addepalli** [comment]
@nitishsharma25 
As per the external bug process we have to log external bugs to MySQL extension. Please reactivate all the bugs and share all the details like VSO, area path details to log an external bug

**Comment by Raghu-addepalli** [comment]
Logged External bug - https://github.com/microsoft/azuredatastudio/issues/22680
We have follow External process for this bug We will act on this bug once external bug is fixed and closed. Please don't close the bug.

**Comment by alanrenmsft** [comment]
@Raghu-addepalli could you please share an example that meets the requirement? I want to see how it is implemented.

**Comment by Raghu-addepalli** [comment]
@nitishsharma25
the section name is not announcing by voice over, when we are using the arrow keys "connection Details" is not getting voice over focus.
it is getting voice over focus and announcing by voice over then it is fine.

**Comment by Raghu-addepalli** [comment]
verified the issue in below version, issue is Fixed Hence closing the Bug and added attachment.

Version details:
Version: 1.45.0-insider (Universal)
Commit: https://github.com/microsoft/azuredatastudio/commit/f7bc38d552e2cfbd7e8cadcf6cb40ed5f7cf9066
Date: 2023-05-09T05:57:43.872Z
VS Code: 1.70.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Darwin x64 22.4.0

[Uploading 197_Fixed (2).zip…]()
